contracts/Strategy.sol

```diff
@@ -489,7 +490,7 @@ contract Strategy is BaseStrategy, DydxFlashloanBase, ICallee {
             uint8 i = 0;
             //position will equal 0 unless we haven't been able to deleverage enough with flash loan
             //if we are not in deficit we dont need to do flash loan
-            while (position > 0) {
+            while (position > 100) {
```

Comment:
this would be the max loss if we want to exit all capital, right?
should it be higher just in case? something like `0.001 ether`?

Reply:
dont want it to be too high because of decimals. like usdc is 6 decimals so 0.001 eth would be alotttt of usdc

Reply:
fair point

Reply:
do we ever need this number to change or be adjustable instead of hardcoding it?

Reply:
I would also double-check whether we need to update this value in the future.